release/obd-kardinal.yaml

```diff
@@ -331,4 +314,63 @@ spec:
       port: 8070
       targetPort: 8070
       protocol: TCP
-      appProtocol: HTTP
\ No newline at end of file
+      appProtocol: HTTP
+
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: gateway
+  annotations:
+    kardinal.dev.service/gateway: "true"
+spec:
+  gatewayClassName: istio
+  listeners:
+    - name: default
+      hostname: "*.app.localhost"
+      port: 8888
+      protocol: HTTP
+      allowedRoutes:
+        namespaces:
+          from: All
+
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http
+  annotations:
+    kardinal.dev.service/route: "true"
+spec:
+  parentRefs:
+    - name: gateway
+  hostnames: ["prod.app.localhost"]
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /
+      backendRefs:
+        - name: frontend
+          port: 80
+
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kardinal.dev.service/ingress: "true"
+  name: ingress
+spec:
+  rules:
+    - host: web.other.localhost
```

Comment:
Do we need this other ingress?

Reply:
not really but works as reference on working with multiple access points and different resources (Gateway and Ingress)